README.md

```diff
@@ -126,6 +134,30 @@ We use [Rollbar](https://rollbar.com) for error monitoring. After every new scri
 
     bin/rollbar-sourcemaps.sh <WP Engine install>
 
+## Troubleshooting
+
+#### Nginx Error
+
+>for project_nginx_1  Cannot create container for service nginx: create .: volume name is too short, names should be at least two alphanumeric characters
+
+Check the version running in the `docker-compose.yml` file within your project directory. We've had issues with version 3.6, try using 3.5 and rebuild to see if this fixes it.
+
+If that doesn't work, check your `.env` file in the root directory. The WP var should equal the path to your WP installation: `WP=./wp` or `WP=./<project-name>`.
+
+#### Mu-plugins Error
```

Comment:
@theasteve I think this is the error @kimpenguin fixed during the demo. LMK. Also caught a spelling mistake, will fix.